interfaces/Referral-App/src/app/components/offer/offer.component.html

```diff
@@ -26,7 +26,10 @@
     <ion-item *ngIf="offer.offerLink">
       <ion-icon name="globe" slot="start"></ion-icon>
       <ion-label>
-        Website: <a href="{{ offer.offerLink }}">{{ offer.offerLink }}</a>
+        Website:
+        <a href="{{ offer.offerLink }}" (click)="logClick('website')">{{
```

Comment:
Maybe good to add the attributes:
```
target="_blank" rel="noopener"
```
to these external links. It's not the best-best-practice perse, but I think that it preserves the 'app-like' feel of the Referral-app a bit better.